crowdsec-docs/unversioned/user_guides/plugins/whm.md

```diff
@@ -0,0 +1,153 @@
+---
+id: whm
+title: WHM plugin
+sidebar_position: 10
+---
+
+The WHM plugin offers a streamlined approach for controlling and monitoring the CrowdSec Security Engine protecting your server directly from your WHM dashboard.  
+Designed specifically for WHM environments, this plugin comes packaged with configurations tailored to enhance your server's security posture efficiently.  
+
+In a few easy steps, this guide walks you through the installation of the WHM plugin, and post-install verification that everything is working fine.
+
+## Installation
+
+### Prerequisites: install CrowdSec
+
+To be able to use this plugin, the first step is to [install CrowdSec Security Engine](https://doc.crowdsec.net/docs/getting_started/install_crowdsec) on your WHM server.
+
+### Download the plugin
+
+First, connect to your WHM server.
```

Comment:
Guess we should say `ssh` instead of connect?

Reply:
"Connect to your WHM server via SSH" ok ?